starfyre/parser.py

```diff
@@ -38,6 +50,8 @@ def __init__(self, component_local_variables, component_global_variables, css, j
             {**self.local_variables, **self.global_variables}
         )
         # populate the dict with the components
+        self.class_name = class_name
+        print(f'class_name={class_name}')
```

Comment:
Do we need this?

Reply:
I believe so. my reasoning is here: https://github.com/sparckles/starfyre/pull/60#discussion_r1263803119

Unless there is no need to inform the user about the file causing the issue, what do you think?